tests/prague/eip7692_eof_v1/tracker.md

```diff
@@ -412,9 +412,9 @@
 - [x] EXTSTATICCALL with input (./eip7692_eof_v1/eip7069_extcall/test_calldata.py)
 - [x] EXTCALL with just enough gas for MIN_RETAINED_GAS and MIN_CALLEE_GAS (./eip7692_eof_v1/eip7069_extcall/test_calls.py::test_eof_calls_min_callee_gas)
 - [x] EXTCALL with not enough gas for MIN_CALLEE_GAS (./eip7692_eof_v1/eip7069_extcall/test_calls.py::test_eof_calls_min_callee_gas)
-- [ ] ADDRESS and CALLER inside EXTCALL (evmone-tests: state_tests/state_transition/eof_calls/extcall_recipient_and_code_address.json)
-- [ ] ADDRESS and CALLER inside EXTDELEGATECALL (evmone-tests: state_tests/state_transition/eof_calls/extdelegatecall_recipient_and_code_address.json)
-- [ ] ADDRESS and CALLER inside EXTSTATICCALL (evmone-tests: state_tests/state_transition/eof_calls/extstaticcall_recipient_and_code_address.json)
+- [x] ADDRESS and CALLER inside EXTCALL (./eip7692_eof_v1/eip7069_extcall/test_calls.py::test_callee_context)
```

Comment:
Have you reviewed the replaced evmone-tests variants? If you replace all the cases from these files you should add them to https://github.com/ethereum/execution-spec-tests/blob/main/converted-ethereum-tests.txt. You will get additional CI feedback with coverage difference.

Reply:
Do the evmone-tests go there? there seem to be none there, it's all `ethereum/tests`

Reply:
Oops. Ignore the comment then.